Sources/HaCWebsite/main.swift

```diff
@@ -1,3 +1,15 @@
 import HaCWebsiteLib
+import DotEnv
+import Foundation
 
-serveWebsite()
+DotEnv.load()
+
+// Check for required env vars
+guard let _ = DotEnv.get("DATA_DIR") else {
+  fatalError("Missing environment variable")
```

Comment:
That would be nice but we want to fail fast (I.e on load) if the environment variable doesn't exist. Config struct sounds like a good option